_includes/documentation.html

```diff
@@ -7,10 +7,10 @@
 		/* Do not record or act on # only hrefs (they serve as menu open/close for empty navigation menu containers). */
 		$('.nav-link').click( function() {
 			var href = $(this).attr('href');
-			var containertest = href.indexOf('#');
-			if(containertest == -1){
+			if(href != "#"){
 				history.pushState({}, null, href);
-				$('#doccontent').animate({scrollTop: 0}, 500).load(href + ' #doccontent > *');
+				$('#doccontent').load(href + ' #doccontent > *');
+				$('html, body').animate({scrollTop: 0}, 500);
```

Comment:
For some reason auto scroll to top of topic broke but separating these two calls out fixed it.